### PR TITLE
docs: fixed module name

### PR DESCRIPTION
**Pull request by helix84**
The same error is also in other modules - there's probably a template somewhere that needs to be fixed.

### CODE REVIEW

**Comment by jirikuncar** [comment]
@helix84 see https://github.com/inveniosoftware/cookiecutter-invenio-module/

**Comment by jirikuncar** [comment]
@helix84 can you add yourself also to authors file?

**Comment by helix84** [comment]
Fixed in inveniosoftware/cookiecutter-invenio-module/pull/59. Added myself to authors here. I hope you don't mind the amend&force-push here. Do you want me to fix up other modules, too, or do you have some kind of automation that could help update them?

**Comment by tiborsimko** [comment]
@helix84 A force push into *feature* branch is OK, but please rebase first against latest `upstream/master`.  We have some semi-automated multi-repository PR update techniques, so that you don't have to do it manually  everywhere...

**Comment by helix84** [comment]
Thanks, rebased. Where can I look to learn more about the automation?

**Comment by jirikuncar** [comment]
@helix84 we are working on polishing our guidelines but most you can find around http://invenio.readthedocs.io/en/latest/processes/developing.html

**Comment by jirikuncar** [comment]
> Added myself to authors here.

@helix84 I can't see the change 😞

**Comment by helix84** [comment]
@jirikuncar Sorry, done.

File: AUTHORS.rst [hunk 1]
```diff
@@ -34,3 +34,4 @@ Media assets management for Invenio.
 - Sami Hiltunen <sami.mikael.hiltunen@cern.ch>
 - Tibor Simko <tibor.simko@cern.ch>
 - Yoan Blanc <yoan@dosimple.ch>
+- Ivan Masar <helix84@centrum.sk>
```
Comment (jirikuncar) [inline AUTHORS.rst]: FYI https://github.com/inveniosoftware/cookiecutter-invenio-module/pull/60

**Pull request merged**